src/test/java/marquez/client/models/JsonGenerator.java

```diff
@@ -121,24 +123,29 @@ private static String newJsonFor(final DbTable dbTable) {
         .put("createdAt", ISO_INSTANT.format(dbTable.getCreatedAt()))
         .put("updatedAt", ISO_INSTANT.format(dbTable.getUpdatedAt()))
         .put("sourceName", dbTable.getSourceName())
-        .put("lastModified", dbTable.getLastModified().map(ISO_INSTANT::format).orElse(null))
+        .put("lastModifiedAt", dbTable.getLastModifiedAt().map(ISO_INSTANT::format).orElse(null))
         .put("description", dbTable.getDescription().orElse(null))
         .toString();
   }
 
   private static String newJsonFor(final Stream stream) {
-    return MAPPER
+    final ArrayNode fields = MAPPER.valueToTree(stream.getFields().orElse(new ArrayList<Field>()));
```

Comment:
Is this correct @wslulciuc ?

Reply:
Yep, the syntax looks right. You'll want to make sure the order of the fields are preserved though. I think that's what's resulting in the test failing. For example, `fields` and `tags` will be the last 2 properties for the Json generated with `JsonGenerator.newJsonFor(Stream)` which differs from the Json returned by the marquezAPI.

Reply:
Done